runtime/queries/b/highlights.scm

```diff
@@ -0,0 +1,187 @@
+(_
+  lhs: (linker) @comment
+  target: (linker) @markup.strong @diff.plus
+  rhs: (linker) @comment)
+
+(linker
+  target: (link_name) @markup.strong @diff.minus) @comment
+
+((identifier) @variable
+  (#set! priority 95))
+
+((identifier) @constant
+  (#set! priority 103)
```

Comment:
Why 103? Seems random.

Reply:
half way between 100 and 105

Reply:
Priorities are a measure of last resort. What actual problem are you trying to solve there?

Reply:
IIRC, when I was testing it out, the color for `@variable` was applied over the color for `@constant`. I wanted behavior similar to Go, but it didn't work without the priority

![image](https://github.com/user-attachments/assets/8fd6af9b-250e-44ae-ad54-a999e582809b)

Reply:
(Also, I am not at all a fan of these convention based "SHOUTING CASE" highlights.)